notebooks/py_scripts/02_Data_Sync.py

```diff
@@ -148,12 +159,16 @@
 #    "custom": {
 #       "kachery_zone": "franklab.default",
 #       "kachery_dirs": {
-#          "cloud": "/your/base/path/.kachery_cloud"
+#          "cloud": "/your/base/path/.kachery-cloud"
```

Comment:
Please mirror this across `00_Setup` and `dj_local_conf_example.json`

Reply:
should `dj_local_conf.json` also have this?

Reply:
I believe `dj_local_conf.json` is what your local instance uses but in spyglass we git ignore it because it is individual to every user. So this should be the case for your local instance but no need to change anything on the spyglass side.

Reply:
perfect, thanks